src/components/DiscountItem/DiscountItem.css

```diff
@@ -0,0 +1,107 @@
+@import url('https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Lato&display=swap');
+*{
+ box-sizing: border-box;
+ padding: 0;
+
+}
+.discount-general-container{
+    max-width: 1920px;
+    margin-left: auto;
+    margin-right: auto;
+}
+.discount-general{
+    max-width: 1214px;
+    height: 665px;
+    margin: auto;
+}
+
+h1{
+    text-align: center;
+    font-size: 42px;
+    line-height: 49px;
+    color: #151875;
+    font-family: 'Josefin Sans'; 
+}
+.three-phrases{
+ display: flex;
+ justify-content:center;
+ gap:30px;
+}
+.three-phrases p{
+    margin-left: 30px;
+}
+.products .n1{
+    font-size: 35px;
+    line-height: 46px;
+    color: #151875;
```

Comment:
please create "_variables.scss" file if you don't have it yet, and declare variables for colors there
after that you've got to use those variables in this file